README.md

```diff
@@ -79,6 +79,31 @@ For example:
               value: "1"
 ```
 
+##### Set Up Update Time Window
+`UPDATE_WINDOW_START` and `UPDATE_WINDOW_STOP` can be used to specify the update time window during updating.
+When `UPDATE_WINDOW_START` and `UPDATE_WINDOW_STOP` is 0:0:0 (default), the feature is disabled.
+
+To enable this feature, go to `bottlerocket-update-operator.yaml`, change `UPDATE_WINDOW_START` and `UPDATE_WINDOW__STOP` to a `hour:minute:second` formate value (UTC (24-hour time notation)). Note that `UPDATE_WINDOW_START` is inclusive and `UPDATE_WINDOW_STOP` is exclusive.
+
+For avoiding stopping in-process node update when update window stops, Bottlerocket update operator reserves 6 mins before `UPDATE_WINDOW_STOP` to finish remaining update circle.
```

Comment:
```suggestion
To avoid stopping an in-process node update when the update window stops, Bottlerocket update operator reserves 6 mins before `UPDATE_WINDOW_STOP` to finish remaining updates.
```